.github/workflows/build.yaml

```diff
@@ -20,6 +20,10 @@ jobs:
       matrix:
         wdk:
           - Microsoft.WindowsWDK.10.0.22621 # NI WDK
+          # Use versions numbers for NuGet WDK packages
+          # - 10.0.26100.1882
+          # - 10.0.26100.1591
```

Comment:
FWIW the varying QFEs here do refer to meaningfully different builds of the WDK now that we've moved to a CD model.  It is probably fine to just use the latest GE CD kit for the matrix rather than run each one.

Reply:
So for a given build number, a user should always be using the latest QFE, right?

> FWIW the varying QFEs here do refer to meaningfully different builds of the WDK now that we've moved to a CD model. It is probably fine to just use the latest GE CD kit for the matrix rather than run each one.

Is this WRT to GE only, or do you think its always fine to just run on the latest QFE for every build number? my intuition here is to have the matrix always run:
* the latest QFE for every build number
* both winget and nuget since they use meaningfully different ways of exposing where its contents are (ie. regkey for winget and vcxproj for nuget, but per my other comment, i think we should piggyback off the ewdks' env var)

Reply:
I believe that prior to Ge there was no meaningful difference between QFEs.  Kits almost exclusively shipped once per release.  I think there was a special case in one release, but the concept of "multiple kits with the same build number but different QFEs, with different content" is new to Ge and CD.